### ISSUE TITLE
TraitsMethodsConflictsInspection on abstract functions

### ISSUE THREAD

**Issue by puggan** [kalessil/phpinspectionsea#1339]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), which version 3.0.1 |
| Language level | PHP 5.6/7.3 |

Question
--
Are TraitsMethodsConflictsInspection supose to trigger on abtract methods in traits?

![Screenshot from 2019-06-17 14-41-34](https://user-images.githubusercontent.com/866668/59605006-04f14080-910e-11e9-833c-ea9093b0d234.png)

```php
<?php

trait A
{
    public function b(): int
    {
        return 3 * $this->f();
    }
    abstract public function f(): int;
}

trait C
{
    public function d(): int
    {
        return 2 * $this->f();
    }
    abstract public function f(): int;
}

class E
{
    public function f(): int
    {
        return 1;
    }
}

class G extends E
{
    use A, C;

    public function test(): int {
        return $this->b() + $this->d();
    }
}

echo (new G())->test();
```

**Comment by kalessil** [comment]
Holy cow, we can have abstracts in traits. Defenetelly false-positive.

**Comment by kalessil** [comment]
Fixed!
